### PR TITLE
Add Koppen Geiger climate zone to station listing

### PR DESCRIPTION
**Pull request by AdamRJensen**
Related to #9

### CODE REVIEW

**Comment by AdamRJensen** [comment]
The documentation build takes significantly longer now (up from ~100 s to now ~200 s).

The kgcPy package is also very poorly documented (th documentation link from PyPI is broken) and hosting of the code is not to be found anywhere online (e.g., GitHub).

**Comment by kandersolar** [comment]
I had the same frustration with that package when I used it some months ago.  By looking at the package's source code, I saw that it is very simple internally; essentially just a wrapper around a lookup table.  However, the `lookupCZ` function reads the lookup table file on every call.  In my application I resorted to extracting the lookup code into a new function, but only reading the table once.  That improved the speed substantially, but of course came with the cost of messy code.

**Comment by AdamRJensen** [comment]
Willing to share that messy code?

**Comment by kandersolar** [comment]
```python
from PIL import Image
Image.MAX_IMAGE_PIXELS = None

import pandas as pd
from importlib import resources

with resources.open_binary('kgcPy', 'kmz_int_reshape.png') as fp:
    kg_img = Image.open(fp)
    kg_img.load()

with resources.open_binary('kgcPy', 'kg_zoneNum.csv') as fp:
    kg_df = pd.read_csv(fp)

def lookupCZ(lat, lon):
    x = round((lon+180)*(kg_img.size[0])/360 - 0.5)
    y = round(-(lat-90)*(kg_img.size[1])/180 - 0.5)
    num = kg_img.getpixel((x, y))
    res = kg_df['kg_zone'].loc[kg_df['zoneNum'] == num]
    return res.values[0]
```

Comparison:
```python
import numpy as np

N = 10
lats = (2 * np.random.rand(N) - 1) * 90
lons = (2 * np.random.rand(N) - 1) * 180

%time kgcz1 = [lookupCZ(lat, lon) for lat, lon in zip(lats, lons)]

import kgcPy

%time kgcz2 = [kgcPy.lookupCZ(lat, lon) for lat, lon in zip(lats, lons)]

for x, y in zip(kgcz1, kgcz2):
    assert x == y
```

```
CPU times: total: 0 ns
Wall time: 3.64 ms
CPU times: total: 1.39 s
Wall time: 1.92 s
```

File: docs/kgcPy2.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
+'''Vectorized version of functions in the kgcPy package'''
```
Comment (kandersolar) [inline docs/kgcPy2.py:1]:
```suggestion
'''Accelerated version of functions in the kgcPy package'''
```

Not "vectorized" in the usual sense; the function still only processes one location at a time.

**Review by kandersolar — APPROVED** [review]
LGTM!

**Comment by AdamRJensen** [comment]
@kandersolar Thanks. I've contacted the authors of the package and hope to implement the changes to the source code fyi.

**Pull request merged**